securedrop_client/gui/main.py

```diff
@@ -197,9 +192,22 @@ def show_conversation_for(self, source: Source, is_authenticated: bool):
 
         self.main_view.set_conversation(conversation_container)
 
-    def set_status(self, message, duration=0):
+    def update_activity_status(self, message: str, duration=0):
```

Comment:
`update_activity_status` is a clearer name than `set_status` which I noticed was accidentally being used to update some error statuses in `logic.py`.